server/services/damsService/generateGetFolderByPathQuery.js

```diff
@@ -0,0 +1,19 @@
+const COLLECTION_WEBSITE_API_FOLDER = "Collection Website API";
+
+function generateGetFolderByPathQuery(objectNumberWithUnderscores) {
```

Comment:
We look for the folder for a specific Object Number using this query - which can return no results if there's no renditions in NetX for the object.